### PR TITLE
Component testing

### PR DESCRIPTION
**Pull request by GovAlberta**

### CODE REVIEW

File: apps/angular-demo-e2e/src/integration/button-component/button-size-components.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+import properties from "../../fixtures/properties.json";
+
+Given(/^cs I am a user of GOA application$/, function () {
+  cy.visit("http://localhost:4200/");
+});
+
+When(/^cs Navigating to Button Type primary component$/, function () {
+  cy.get("[ng-reflect-label='Button']").click();
```
Comment (SamKothari1) [inline apps/angular-demo-e2e/src/integration/button-component/button-size-components.ts:9]: You could also go with: [label=Button]
cy.get('[label=Button]').click();

File: apps/angular-demo-e2e/src/integration/button-component/button-size-components.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+});
+
+function remToPx(rem) {
+  return rem.replace("rem", "") * 16 + "px";
+}
+
+//function to convery hex to rgb
+// return rgb(r, g, b)
+function hexToRgb(hex) {
+  var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
```
Comment (SamKothari1) [inline apps/angular-demo-e2e/src/integration/button-component/button-size-components.ts:46]: Please use const instead of var

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**